memcache/memcache.go

```diff
@@ -70,10 +70,32 @@ const (
 	// DefaultMaxIdleConns is the default maximum number of idle connections
 	// kept for any single address.
 	DefaultMaxIdleConns = 2
+
+	base64MetaFlag                = "b"
+	casTokenResponseMetaFlag      = "c"
+	compareCasValueTokenMetaFlag  = "C"
+	setClientFlagsToTokenMetaFlag = "F"
+	invalidateMetaFlag            = "I"
+	returnKeyAsTokenMetaFlag      = "k"
+	opaqueTokenMetaFlag           = "O"
+	noReplySemanticsMetaFlag      = "q"
+	updateTTLTokenMetaFlag        = "T"
+	modeTokenMetaFlag             = "M"
+	ttlResponseMetaFlag           = "t"
 )
 
 const buffered = 8 // arbitrary buffered channel size, for readability
 
+type MetaSetMode string
+
+const (
+	Add     MetaSetMode = "E"
+	Append  MetaSetMode = "A"
+	Prepend MetaSetMode = "P"
+	Replace MetaSetMode = "R"
+	Set     MetaSetMode = "S"
```

Comment:
Probably not, I also added it for completeness :P I don't know if there is ever a use case for this but considering it is a valid flag in memcached, there might be some scenario we might not be considering where it could be useful.